src/components/layout/Layout.jsx

```diff
@@ -15,12 +15,12 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import { AppContext } from '../app-context';
 
-import EdXLogo from '../../images/edx-logo.svg';
+import DefaultLogo from '../../images/logo.svg';
```

Comment:
We'll want to remove `images/logo.svg` from this repo as logos should no longer be static assets in individual repos in favor of the shared logo URLs hosted on our CDN. The logo urls are now coming from environment variables, e.g. `LOGO_URL=https://edx-cdn.org/v3/prod/logo.svg`.

frontend-platform@1.8.0 exposes these logo urls in its configuration. See https://openedx.atlassian.net/wiki/spaces/AC/pages/2047345474/Logo+Flip+Plan+and+Instructions for more details on this. It looks like we've installed frontend-platform@1.8.0 in this repo but are not using it. If frontend-platform's way of exposing these logo URL settings doesn't work in this repo for whatever reason, falling back to `process.env` should suffice.

Note that because this is used in a Gatsby context which only exposes environment variables with a `GATSBY_` prefix, we'll need to ensure these environment variables are accessible by frontend-app-learner-portal-programs: https://github.com/edx/frontend-app-learner-portal-programs/blob/master/gatsby-config.js#L40.